{"uid":"5c13343a6c42de42","name":"test_site_title","fullName":"tests.test_demo.Ayuda#test_site_title","historyId":"2de973a726f6108f7c3b207b90ba2fc4","time":{"start":1657905743103,"stop":1657905753190,"duration":10087},"status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":true,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_unittest_setUpClass_fixture_Ayuda","time":{"start":1657905743100,"stop":1657905743101,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"testStage":{"status":"passed","steps":[],"attachments":[{"uid":"85c4f83db0b630ed","name":"stdout","source":"85c4f83db0b630ed.txt","type":"text/plain","size":8}],"parameters":[],"stepsCount":0,"attachmentsCount":1,"shouldDisplayMessage":false,"hasContent":true},"afterStages":[{"name":"_unittest_setUpClass_fixture_Ayuda::0","time":{"start":1657905755792,"stop":1657905755792,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"labels":[{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_demo"},{"name":"subSuite","value":"Ayuda"},{"name":"host","value":"fv-az36-129"},{"name":"thread","value":"2848-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_demo"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":7,"skipped":0,"passed":2,"unknown":0,"total":9},"items":[{"uid":"1a3f9e0f7bfd8273","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/32//#testresult/1a3f9e0f7bfd8273","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x561f4c384b13 <unknown>\n#1 0x561f4c18b688 <unknown>\n#2 0x561f4c1aff1f <unknown>\n#3 0x561f4c1ab5aa <unknown>\n#4 0x561f4c1e664a <unknown>\n#5 0x561f4c1e07a3 <unknown>\n#6 0x561f4c1b60ea <unknown>\n#7 0x561f4c1b7225 <unknown>\n#8 0x561f4c3cc2dd <unknown>\n#9 0x561f4c3d02c7 <unknown>\n#10 0x561f4c3b622e <unknown>\n#11 0x561f4c3d10a8 <unknown>\n#12 0x561f4c3aabc0 <unknown>\n#13 0x561f4c3ed6c8 <unknown>\n#14 0x561f4c3ed848 <unknown>\n#15 0x561f4c407c0d <unknown>\n#16 0x7fd062ac8609 <unknown>","time":{"start":1657905359491,"stop":1657905362602,"duration":3111}},{"uid":"ec5afd89a43d633","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/31//#testresult/ec5afd89a43d633","status":"passed","time":{"start":1657904722269,"stop":1657904731203,"duration":8934}},{"uid":"54c1bdd72f8fffd2","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/30//#testresult/54c1bdd72f8fffd2","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55c02ece5b13 <unknown>\n#1 0x55c02eaec688 <unknown>\n#2 0x55c02eb10f1f <unknown>\n#3 0x55c02eb0c5aa <unknown>\n#4 0x55c02eb4764a <unknown>\n#5 0x55c02eb417a3 <unknown>\n#6 0x55c02eb170ea <unknown>\n#7 0x55c02eb18225 <unknown>\n#8 0x55c02ed2d2dd <unknown>\n#9 0x55c02ed312c7 <unknown>\n#10 0x55c02ed1722e <unknown>\n#11 0x55c02ed320a8 <unknown>\n#12 0x55c02ed0bbc0 <unknown>\n#13 0x55c02ed4e6c8 <unknown>\n#14 0x55c02ed4e848 <unknown>\n#15 0x55c02ed68c0d <unknown>\n#16 0x7f0a90886609 <unknown>","time":{"start":1657904306984,"stop":1657904310433,"duration":3449}},{"uid":"dbe9e4bd31b5095c","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/29//#testresult/dbe9e4bd31b5095c","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55af1bf16b13 <unknown>\n#1 0x55af1bd1d688 <unknown>\n#2 0x55af1bd41f1f <unknown>\n#3 0x55af1bd3d5aa <unknown>\n#4 0x55af1bd7864a <unknown>\n#5 0x55af1bd727a3 <unknown>\n#6 0x55af1bd480ea <unknown>\n#7 0x55af1bd49225 <unknown>\n#8 0x55af1bf5e2dd <unknown>\n#9 0x55af1bf622c7 <unknown>\n#10 0x55af1bf4822e <unknown>\n#11 0x55af1bf630a8 <unknown>\n#12 0x55af1bf3cbc0 <unknown>\n#13 0x55af1bf7f6c8 <unknown>\n#14 0x55af1bf7f848 <unknown>\n#15 0x55af1bf99c0d <unknown>\n#16 0x7f158a993609 <unknown>","time":{"start":1657903183177,"stop":1657903186417,"duration":3240}},{"uid":"6b64e6a8b72ec266","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/28//#testresult/6b64e6a8b72ec266","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55b624426b13 <unknown>\n#1 0x55b62422d688 <unknown>\n#2 0x55b624251f1f <unknown>\n#3 0x55b62424d5aa <unknown>\n#4 0x55b62428864a <unknown>\n#5 0x55b6242827a3 <unknown>\n#6 0x55b6242580ea <unknown>\n#7 0x55b624259225 <unknown>\n#8 0x55b62446e2dd <unknown>\n#9 0x55b6244722c7 <unknown>\n#10 0x55b62445822e <unknown>\n#11 0x55b6244730a8 <unknown>\n#12 0x55b62444cbc0 <unknown>\n#13 0x55b62448f6c8 <unknown>\n#14 0x55b62448f848 <unknown>\n#15 0x55b6244a9c0d <unknown>\n#16 0x7fd9e9b70609 <unknown>","time":{"start":1657896732163,"stop":1657896733533,"duration":1370}},{"uid":"3fd112bad02d1f61","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/27//#testresult/3fd112bad02d1f61","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: 'chromedriver' executable needs to be in PATH. Please see https://chromedriver.chromium.org/home","time":{"start":1657839031623,"stop":1657839031624,"duration":1}},{"uid":"2463d8e9947a801c","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/26//#testresult/2463d8e9947a801c","status":"broken","statusDetails":"NameError: name 'driver' is not defined","time":{"start":1657838860866,"stop":1657838860867,"duration":1}},{"uid":"ae55eb0012aa2a06","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/25//#testresult/ae55eb0012aa2a06","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: 'chromedriver' executable needs to be in PATH. Please see https://chromedriver.chromium.org/home","time":{"start":1657838605845,"stop":1657838605846,"duration":1}}]},"tags":[]},"source":"5c13343a6c42de42.json","parameterValues":[]}